eng/Directory.Build.Common.props

```diff
@@ -119,6 +121,10 @@
     <RequiredTargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net461;netcoreapp3.1;net5.0</RequiredTargetFrameworks>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(IsMgmtSubLibrary)' == 'true' and !$(MSBuildProjectName.Equals('Azure.ResourceManager.Tests'))">
```

Comment:
Should we just exclude test projects? It seems like you opted out a bunch of test projects directly.